src/index.ts

```diff
@@ -806,161 +842,161 @@ export class SunsynkPowerFlowCard extends LitElement {
             </g>
 
             <a href="#" @click=${(e) => this.handlePopup(e, config.entities.use_timer_248)}>
-              <svg xmlns="http://www.w3.org/2000/svg" id="timer" x="46%" y="${useautarky != false ? "60.5%" : "65%"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj26.state == 'on' && usetimer !== false ? '' : 'none'}" fill="${inverter_colour}" d="M15 1H9v2h6V1zm-4 13h2V8h-2v6zm8.03-6.61l1.42-1.42c-.43-.51-.9-.99-1.41-1.41l-1.42 1.42A8.962 8.962 0 0 0 12 4c-4.97 0-9 4.03-9 9s4.02 9 9 9a8.994 8.994 0 0 0 7.03-14.61zM12 20c-3.87 0-7-3.13-7-7s3.13-7 7-7s7 3.13 7 7s-3.13 7-7 7z"/></svg>
-              <svg xmlns="http://www.w3.org/2000/svg" id="timer_off" x="46%" y="${useautarky != false ? "60.5%" : "65%"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj26.state == 'off' && usetimer !== false ? '' : 'none'}" fill="${inverter_colour}" d="m19.95 17.15l-1.5-1.5q.275-.675.413-1.337T19 13q0-2.9-2.05-4.95T12 6q-.6 0-1.275.125t-1.4.4l-1.5-1.5q.95-.5 2.012-.763T12 4q1.5 0 2.938.5t2.712 1.45l1.4-1.4l1.4 1.4l-1.4 1.4q.95 1.275 1.45 2.713T21 13q0 1.05-.263 2.087t-.787 2.063ZM13 10.2V8h-2v.2l2 2Zm6.8 12.4l-2.4-2.4q-1.2.875-2.588 1.338T12 22q-1.85 0-3.488-.713T5.65 19.35q-1.225-1.225-1.938-2.863T3 13q0-1.5.463-2.888T4.8 7.6L1.4 4.2l1.4-1.4l18.4 18.4l-1.4 1.4ZM12 20q1.05 0 2.05-.325t1.875-.925L6.2 9.025q-.6.875-.9 1.875T5 13q0 2.9 2.05 4.95T12 20ZM9 3V1h6v2H9Zm2.075 10.875Zm2.825-2.8Z"/></svg>
-              <text id="timer_text_on"x="50%" y="${useautarky != false ? "63.5%" : "68%"}" class="st3 left-align" display="${stateObj26.state == 'on' && usetimer !== false ? '' : 'none'}" fill="${inverter_colour}">${localize('common.timer_on')}</text>
-              <text id="timer_text_off"x="50%" y="${useautarky != false ? "63.5%" : "68%"}" class="st3 left-align" display="${stateObj26.state == 'off' && usetimer !== false ? '' : 'none'}" fill="${inverter_colour}">${localize('common.timer_off')}</text>
+              <svg xmlns="http://www.w3.org/2000/svg" id="timer" x="210" y="${useautarky != 'no' ? "232" : "249"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj26.state == 'on' && usetimer !== 'no' ? '' : 'none'}" fill="${inverter_colour}" d="M15 1H9v2h6V1zm-4 13h2V8h-2v6zm8.03-6.61l1.42-1.42c-.43-.51-.9-.99-1.41-1.41l-1.42 1.42A8.962 8.962 0 0 0 12 4c-4.97 0-9 4.03-9 9s4.02 9 9 9a8.994 8.994 0 0 0 7.03-14.61zM12 20c-3.87 0-7-3.13-7-7s3.13-7 7-7s7 3.13 7 7s-3.13 7-7 7z"/></svg>
```

Comment:
These attributes (useautarky and usetimer) should be false not no?

Reply:
So this one is slightly different, as the option `config.inverter?.autarky` has strings in it, the type needs to stay a string, therefore, using no is ok. 

the only way around that would be to add an option `show_autarky` and then have `autarky_mode` which had `power`/`energy`

im not sure it would be worth adding a new option but it could be done easily enough to make everything follow a standard.

Reply:
OK understood. No not worth it.